templates/zerver/config_error.html

```diff
@@ -80,6 +80,12 @@ <h1 class="lead">Configuration error</h1>
                         {% endif %}
                     {% endif %}
 
+                    {% if saml_error %}
+                    <p>
+                        SAML authentication is eitheir not enabled or misconfigured. See setup instructions
+                        in <code>/etc/zulip/settings.py</code>.
```

Comment:
I think we want to move the main SAML setup docs to a section on `docs/production/authentication-methods.md` and link there to match our standard approach for new settings with complex instructions.

Reply:
Put a link there:
```

                    {% if saml_error %}
                    <p>
                        SAML authentication is either not enabled or misconfigured. Have a look at
                        our <a href="https://zulip.readthedocs.io/en/latest/production/authentication-methods.html#SAML">setup guide</a>.
                    </p>
                    {% endif %}
```